src/features/homePage/HomePage.tsx

```diff
@@ -1,9 +1,42 @@
 import './HomePage.scss';
 import { NavLink } from 'react-router-dom';
+import { useEffect, useState } from 'react';
 import { Slider } from './components/Slider/Slider';
 import { BannerSlider } from './components/BannerSlider/BannerSlider';
 
 export const Home = () => {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Why didn't you use isLoading or error state? We don't have enough time, so for now, I'll merge it and fix it in my next pull request